tools/circexplorer/circexplorer.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="circexplorer" name="CIRCexplorer" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="2.0.14">tophat</requirement>
```

Comment:
You can add `<requirement type="package" version="0.6.2">docopt</requirement>` to get the python package. It is included in conda.
Please also change the requirement to tophat 2.1.0 to match the conda version.